core/renderers/common/drawer.js

```diff
@@ -10,29 +10,30 @@
  */
 'use strict';
 
-goog.provide('Blockly.blockRendering.Drawer');
+goog.module('Blockly.blockRendering.Drawer');
+goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.blockRendering.RenderInfo');
-goog.require('Blockly.blockRendering.Row');
-goog.require('Blockly.blockRendering.Types');
-goog.require('Blockly.utils.svgPaths');
+const Row = goog.require('Blockly.blockRendering.Row');
```

Comment:
The requires here should be ordered based on the [style guide](https://google.github.io/styleguide/jsguide.html#file-goog-require), mainly alphabetize (and don't separate the `requireType` from `require`, based on example).